backend/app/Models/User.php

```diff
@@ -76,4 +77,10 @@ public function userTransactions()
     {
         return $this->hasMany(Transaction::class);
     }
+
+    public function sendPasswordResetNotification($token)
+    {
+        $url = '{{api}}/reset-password?token=' . $token;
```

Comment:
```suggestion
        $url = '?{{api}}?/reset-password?token=' . $token;
```
why are you putting the postman variable here? do we expect users to have the same postman setup?